templates/vue/src/components/Lightbox/media/ActivityMedia/index.vue

```diff
@@ -71,20 +110,35 @@ export default {
       required: false,
       default: "",
     },
+    initialType: {
+      type: String,
+      required: true,
+    },
   },
   data() {
     return {
       activeQuestionIndex: 0,
-      showCompletionScreen: false,
+      state: "",
     }
   },
   computed: {
-    ...mapGetters(["getAnswers"]),
+    ...mapGetters(["getAnswers", "getQuestion", "getNode"]),
+    questionNode() {
+      return this.initialType === states.ACTIVITY
+        ? this.node
+        : this.getNode(this.node.typeData.activityId)
```

Comment:
Can you remind me what an activityId is exactly? I do not remember having this in the data structure. I thought we only have node Ids that are for each activity.